lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -1167,6 +1168,7 @@ def test_certificate_custom_language_template_with_org_mode_and_course_key(
         This test should check template matching when org={org}, course={course}, mode={mode}.
         """
         DarkLangConfig(released_languages='es-419, fr', changed_by=self.user, enabled=True).save()
+        self.addCleanup(cache.clear)
```

Comment:
Adding `CacheIsolationMixin` to the class might be simpler in cases like this where multiple test cases add stuff to the cache.  Either way is fine, though.